aya/src/sys/netlink.rs

```diff
@@ -117,7 +118,7 @@ pub(crate) unsafe fn netlink_qdisc_attach(
         nlmsg_seq: 1,
     };
     req.tc_info.tcm_family = AF_UNSPEC as u8;
-    req.tc_info.tcm_handle = 0; // auto-assigned, if not provided
+    req.tc_info.tcm_handle = handle; // auto-assigned, if not provided
```

Comment:
Maybe to be consistent with your comment in `netlink_qdisc_detach`:

```suggestion
    req.tc_info.tcm_handle = handle; // auto-assigned, if zero
```

Reply:
Will do.

Reply:
@vadorovsky, thanks for the suggestions.